tests/src/test/scala/org/dhallj/tests/acceptance/AcceptanceSuites.scala

```diff
@@ -25,6 +25,9 @@ class TypeCheckingSimpleSuite extends CachingTypeCheckingSuite("type-inference/s
 class TypeCheckingUnitSuite extends CachingTypeCheckingSuite("type-inference/success/unit")
 class TypeCheckingRegressionSuite extends TypeCheckingSuite("type-inference/success/regression")
 class TypeCheckingOtherSuite extends TypeCheckingSuite("type-inference/success") {
+  //TODO prelude is WAY too slow
```

Comment:
What's way too slow? To be honest I'd rather hold off on this change for now and keep it easy to make sure we're correctly type-checking the unnormalized prelude in CI, etc.

Reply:
The prelude test is what was hanging and aborting the CI builds. We can do, it just means we don't conform to the spec and will have to disable one or two other tests and I thought most of this was covered in the `Prelude` suites anyway

Reply:
Okay, let's go ahead and merge this after the `Prelude` name change revert and I'll take a closer look later at making sure we can run this.

Reply:
👍 I'm going to have a look into the Haskell implementation to see if it does any optimization (caching type-checking results, etc)